src/compilationSamples/__tests__/__snapshots__/expectSaga.test.ts.snap

```diff
@@ -11,17 +11,17 @@ Array [
     Property 'text' is missing in type 'CountReducerState' but required in type '{ text: string; }'.",
   },
   Object {
-    "character": 54,
+    "character": 53,
     "fileName": "expectSaga-test.ts",
     "line": 55,
     "message": "Argument of type '5' is not assignable to parameter of type 'boolean'.",
   },
   Object {
-    "character": 19,
+    "character": 20,
     "fileName": "expectSaga-test.ts",
     "line": 56,
-    "message": "Argument of type 'Response' is not assignable to parameter of type 'string | Request'.
-  Type 'Response' is missing the following properties from type 'Request': context, method, redirect, referrer, and 5 more.",
+    "message": "Argument of type 'Action<{ notText: string; }>' is not assignable to parameter of type 'Action<{ text: string; }>'.
```

Comment:
@HenriBeck Wondering why this error message got fixed, since there wasn't a change to the interface of `whenDispatched`.

Reply:
TypeScript doesn't find all violations here. I added some variations of the test to capture the other errors as well.